modules/docs/arrow-docs/docs/docs/quickstart/libraries/README.md

```diff
@@ -10,178 +10,246 @@ Arrow is a modular set of libraries that build on top of each other to provide i
 
-For function objects the library provides composition, currying, partial application, memoization, pipe operator, complement for predicates, and several more helpers.
+Typeclasses: [`Alternative`]({{ '/docs/arrow/typeclasses/alternative/' | relative_url }}), [`Bimonad`]({{ '/docs/arrow/typeclasses/bimonad/' | relative_url }}), [`Inject`]({{ '/docs/typeclasses/inject/' | relative_url }}), [`Reducible`]({{ '/docs/arrow/typeclasses/reducible/' | relative_url }}), [`Traverse`]({{ '/docs/arrow/typeclasses/traverse/' | relative_url }}), [`Applicative`]({{ '/docs/arrow/typeclasses/applicative/' | relative_url }}), [`Comonad`]({{ '/docs/arrow/typeclasses/comonad/' | relative_url }}), [`Eq`]({{ '/docs/arrow/typeclasses/eq/' | relative_url }}), [`Monad`]({{ '/docs/arrow/typeclasses/monad/' | relative_url }}), [`Monoid`]({{ '/docs/arrow/typeclasses/monoid/' | relative_url }}), [`Semigroup`]({{ '/docs/arrow/typeclasses/semigroup/' | relative_url }}), [`ApplicativeError`]({{ '/docs/arrow/typeclasses/applicativeerror/' | relative_url }}), [`Foldable`]({{ '/docs/arrow/typeclasses/foldable/' | relative_url }}), [`MonoidK`]({{ '/docs/arrow/typeclasses/monoidk/' | relative_url }}), [`SemigroupK`]({{ '/docs/arrow/typeclasses/semigroupk/' | relative_url }}), [`Bifoldable`]({{ '/docs/arrow/typeclasses/bifoldable/' | relative_url }}), [`Functor`]({{ '/docs/arrow/typeclasses/functor/' | relative_url }}), [`MonadError`]({{ '/docs/arrow/typeclasses/monaderror/' | relative_url }}), [`Order`]({{ '/docs/arrow/typeclasses/order/' | relative_url }}), [`Show`]({{ '/docs/arrow/typeclasses/show/' | relative_url }}), [`TraverseFilter`]({{ '/docs/arrow/typeclasses/traversefilter/' | relative_url }}), [`FunctorFilter`]({{ '/docs/arrow/typeclasses/functorfilter/' | relative_url }}), [`MonadFilter`]({{ '/docs/arrow/typeclasses/monadfilter/' | relative_url }}), [`MonadCombine`]({{ '/docs/arrow/typeclasses/monadcombine/' | relative_url }}), `Composed`
 
-For collections, arrow-syntax provides `firstOption`, tail, basic list traversal, and tuple addition.
+### arrow-optics
 
-### arrow-typeclasses
+```groovy
+dependencies {
+    compile "io.arrow-kt:arrow-optics:$arrow_version"
+}
+```
 
-{:.intermediate}
-intermediate
+Optics is the functional way of handling immutable data and collections in a way that's boilerplate free and efficient.
 
-All the basic [typeclasses]({{ '/docs/typeclasses/intro' | relative_url }}) that can compose into a simple program.
+It can be used alongside annotation processing to generate [simple DSLs]({{ '/docs/optics/dsl/' | relative_url }}) that read like imperative code.
```

Comment:
"imperative code" isn't accurate since there is no alternative. Imperative code doesn't allow for complex nested traversal updates of mutable or immutable structures.

Not sure what to suggest. Perhaps something about what it does instead of what it looks like?

Reply:
What do you have in mind,  I solely merged the text description fro, the older version

Reply:
If you prefer we can also delete the description. And leave it as a dependency import section, maybe link a good example what optics can do

Reply:
Nono, we should definitely have a good short description here to promote libraries.

Reply:
```suggestion
Arrow Optics offers a way of declaring how to focus deeply into immutable structure without boilerplate. It also offers an [Optics DSL]({{ '/docs/optics/dsl/' | relative_url }})  to elegantly describe complex use-cases in an elegant and simple manner without requiring to understand the underlying theory.
```